src/Api/Model/Statistics/ReleaseInfo.ts

```diff
@@ -0,0 +1,9 @@
+import { ItemsPerMonth } from './StatisticsResponse'
+
+export type ReleaseInfo = {
+  readonly releasesPerMonth: ItemsPerMonth[]
```

Comment:
see suggestions below

```suggestion
  readonly releasesPerMonth: ItemsPerMonth
```